Geometry_Engine/Query/IsContaining.cs

```diff
@@ -709,6 +709,168 @@ public static bool IsContaining(this Cuboid cuboid, IGeometry geometry, bool acc
+        [Input("tolerance", "Distance tolerance to use to determine intersections.")]
+        [Output("isContaining", "True if the point is contained within the bounds of the panels, false if it is not for each point provided.")]
+        public static bool IsContaining(this List<Polyline> closedVolume, Point point, bool acceptOnEdges = false, double tolerance = BH.oM.Geometry.Tolerance.Distance)
+        {
+            List<Point> pointLists = new List<Point>() { point };
+            return IsContaining(closedVolume, pointLists, acceptOnEdges, tolerance).First();
+        }
+
+        /***************************************************/
+
+        [Description("Checks if each of a list of points lies within a collection of polylines describing a closed volume using their primitive planes and bounds.")]
+        [Input("closedVolume", "A collection of Polylines outlining each of the faces of the closed volume to check with.")]
+        [Input("points", "The points to check to see if it is contained within the bounds of the panels.")]
+        [Input("acceptOnEdges", "Decide whether to allow the point to sit on the edge of the panel, default false.")]
+        [Input("tolerance", "Distance tolerance to use to determine intersections.")]
+        [Output("isContaining", "True if the point is contained within the bounds of the panels, false if it is not for each point provided.")]
+        public static List<bool> IsContaining(this List<Polyline> closedVolume, List<Point> points, bool acceptOnEdges = false, double tolerance = BH.oM.Geometry.Tolerance.Distance)
+        {           
+            List<List<Point>> pointLists = points.Select(x => new List<Point>() { x }).ToList();
+            return IsContaining(closedVolume, pointLists, acceptOnEdges, false, tolerance);
+        }
+
+        /***************************************************/
+
+        [Description("Checks if a list of points lies within a collection of polylines describing a closed volume using their primitive planes and bounds.")]
+        [Input("closedVolume", "A collection of Polylines outlining each of the faces of the closed volume to check with.")]
+        [Input("points", "The points to check to see if it is contained within the bounds of the panels.")]
+        [Input("acceptOnEdges", "Decide whether to allow the point to sit on the edge of the panel, default false.")]
+        [Input("acceptPartialContainment", "Decide whether to allow some of the points to sit outside the panels as long as at least one is within them.")]
+        [Input("tolerance", "Distance tolerance to use to determine intersections.")]
+        [Output("isContaining", "True if the points are contained within the bounds of the panels, false if they are not.")]
+        public static bool IsContaining(this List<Polyline> closedVolume, List<Point> points, bool acceptOnEdges = false, bool acceptPartialContainment = false, double tolerance = BH.oM.Geometry.Tolerance.Distance)
```

Comment:
Do we need both overloads taking a list of points, with `acceptPartialContainment` and without?

Reply:
Yes, one checks all points for a single containment value, the other checks a list of points for individual containment and returns a list of result values.